### ISSUE TITLE
Ключ "key_skills" в методе поиска вакансий https://api.hh.ru/vacancies

### ISSUE THREAD

**Issue by JustickDM** [hhru/api#653]
Добавьте, плиз)))

**Comment by JustickDM** [comment]
Нет, Вы не поняли, я не хочу осуществлять поиск по ключевым навыкам, я хочу получить список ключевых навыков в методе https://api.hh.ru/vacancies, в модели не предусмотрено это поле и для того, чтобы получить список ключевых навыков всех полученных вакансий, требуется осуществить лишние 4 тысячи запросов. issue считаю незакрытой.

**Comment by JustickDM** [comment]
Ясно, манагеры просто положили болт на адекватные вопросы, кривожопное апи с моделями, так еще и саппорта нет.

**Comment by tpasichnik** [comment]
Добрый день. 

Приношу свои извинения за длительное ожидание.
Ваше пожелание по отображению атрибута "key_skills" в методе "GET /vacancies" будет учтено.

**Comment by VAU66** [comment]
А что в итоге? Будет учтено. Проблема актуальна. Количество запросов ограничено в  добавок.

**Comment by tpasichnik** [comment]
Добрый день.

Разработчики рассмотрели ваш запрос. К сожалению, выполнить ваше пожелание не сможем по техническим соображениям.
